nss-day-cohort-48/rare-client-boastful-bulldogs
# Delete Post Functionality

Added edit and delete buttons to logged-in user's posts. Added redirects to nav links IF user is not logged in. Also added a touch of styling.

## Steps to test
1. git fetch --all
2. git checkout anm-delete-post
3. need to pull server branch and run debugger
4. serve
5. verify you can't use my post, tag manager, new post links IF not logged in
6. verify if you're logged in that you see edit and delete buttons next to your authored posts.
